### DISCUSSION TITLE
Welcome to the Medjed project

### DISCUSSION THREAD

**Discussion by NemoAndrea** [openMLA/Medjed#1] [Announcements]
This GitHub discussions is set up for 2 primary purposes:

**1. Add logs of hardware developments (new versions, ideas, prototypes)
2. For community questions/ideas**

## ❓ Discussions vs Wiki

Keep in mind that the more 'ironed out' ideas and concepts will be listed in the [📖 project wiki](https://github.com/openMLA/Medjed/wiki). The hardware logs may be as simple as a single image showing a minor edit or a new concept drawing.

## 📣 Community comments

Feel free to comment anything you like. All things are welcome:
1. Questions about why certain parts were picked
4. Suggestions about minor revisions
5. Suggestions about major alternative design options
6. Questions about details of the project
7. New ideas that could be useful for the project
8. Errors or doubts about some of the documentation (e.g. errors in math or assumptions)

**Comment by AdrianPhil** [comment]
Hi, I'm Adrian and I'm taking the Open Hardware Academy course.
This seems a very interesting project and I apologize for not knowing more about this subject and the dumb questions:

Would this do the coating and repeat exposures or the coating is done separately and this just does the exposure.
Would it be simpler to move the UV beam instead of moving the plate?
Or maybe use a mask that one can change digitally? Like the LCD mask in most of today's TVs?
I had and still have the intention to do a project that needs XYZ motion. After considering for a while I  think it would be counter productive to create the device for the XYZ positioning because it would be hard to match the price tag of an existing cheap system. So I'm thinking to just use a 200$ COTS and work on the head and the software part. Would that be something that could also work in this case?

**Reply by NemoAndrea** [reply]
No worries, asking questions is greatly appreciated :).  

The Medjed system would not apply the photoresist layer to the substrate. This would typically be done in a spincoater. There are some open source designs out there, but none that are quite polished enough for routine use imo. 

Moving the beam might be possible at lower resolution (e.g. 50um+) but for the higher resolution the system targets this is not feasible. Laser printers or e.g. the [hexastorm prism laser scanner](https://hackaday.io/project/21933-prism-laser-scanner) walk the beam quite  large distance, but are all fairly large pixel size.

A digital micromirror array is digitally controlled, but a monochrome LCD is used for UV patterning in e.g. resin 3D printers. There it is not used to shape the beam before projection but rather as a shadow mask. They are not very practical for small scale work as the physical pixel size is rather large (50um). That and they have a pretty limited lifetime due to poor UV transmission.

For the motion system, the XY does not need to be very precise, but it must be fitted with high resolution encoders. See [this wiki item for more](https://github.com/openMLA/.github/wiki/On-XY-precision-and-image-stitching). For the Z-focus, it would need something that can step with at most 5um step size and integrate with some sort of autofocus feedback for best results.

But this is all because the Medjed system is to become a reliable automated tool, and so a minimal version of a UV patterning system is a lot simpler. If you dont need image stitching, reliable focus and dont mind fumbling around with image conversion you could take a standard bulb projector and swap out the projector lens for a microscope objective and produce pretty good results with trials.

If you are not interested in sub 10um resolution (e.g. pcb fab), a simple UV resin printer will probably be the best way to go. I have a [little PCB](https://github.com/openMLA/photon-ultra-controller) (not fully tested yet) that makes this workflow easier.
